code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/DatabaseManager.java

```diff
@@ -10,14 +11,15 @@
  */
 public class DatabaseManager {
     private static DatabaseManager instance; // Singleton instance of DatabaseManager
-    private final FirebaseDatabase database; // Instance of FirebaseDatabase
+    private final FirebaseFirestore database; // Instance of FirebaseDatabase
```

Comment:
Is @Aryan-Rusia-2 aware of this change?

Reply:
So, to make it clear, there are two type of database present. One is realtime database and another is firestore database. I was using realtime for notifications so I added that in the DatabaseManager. But I think for the other stuff, firestore database would be better so it's fine to use that. I would request Adrien to write new methods specific to Firestore database without altering the existing Realtime Database methods, as I'm using those.

Reply:
I will implement this in a separate branch tonight

Reply:
Okay David, sounds good. I can correct this as well, but after tomorrow's midterm 🙂, if that works for you!

Reply:
i edited the code to add a firebase firestore database along with the firebase firebase database